src/APICalls.js

```diff
@@ -0,0 +1,30 @@
+const checkForErrors = response => {
+  if(!response.ok) {
+    throw new Error (response.status)
+  } else {
+    return response.json()
+  }
+}
+
+// export const fetchSongData = () => {
+//   // console.log([...fetchAllSongs(), ...fetchAllGenres()])
+//   // return [...fetchAllSongs(), ...fetchAllGenres()]
+//   console.log(fetchAllGenres())
+// }
+
+
+export const fetchAllSongs = () => {
+  return fetch('https://carryokay-server.herokuapp.com/songs')
+    .then(checkForErrors)
+    .then(data => {
+      return data
```

Comment:
I would use the localhost8080 for these right now, because it has the most up to date data including the genre data

Reply:
Got it!